code/website/index.html

```diff
@@ -215,7 +215,7 @@ <h3 id="caseStudies">Case Studies</h3>
 $if(docsUrl)-$
   <h3 id="haskDoc">Haddock Documentation</h3>
   <p>
-    The current <a href="$docsUrl$">Haddock documentation</a> for the Drasil framework.
+    The current <a href="$docsUrl$">Haddock documentation</a> for the Drasil framework. $if(fullDocsUrl)-$A variant with fully exposed modules is also available <a href="$fullDocsUrl$">here</a>.$-endif$
```

Comment:
Please make "fully exposed modules" the link words. "here" links are an anti-pattern.